routes/v1.js

```diff
@@ -248,22 +263,22 @@ api.declare({
 /** Delete hook definition**/
 api.declare({
   method:       'delete',
-  route:        '/hooks/:hookGroup/:hookId',
+  route:        '/hooks/:hookGroupId/:hookId',
   name:         'removeHook',
   idempotent:   true,
-  scopes:       [["hooks:modify-hook:<hookGroup>/<hookId>"]],
+  scopes:       [["hooks:modify-hook:<hookGroupId>/<hookId>"]],
   title:        'Delete a hook',
   description: [
-    "Remove a hook definition."
+    "This endpoint will remove a hook definition."
   ].join('\n')
 }, async function(req, res) {
-  var groupId = req.params.hookGroup;
+  var hookGroupId = req.params.hookGroupId;
   var hookId = req.params.hookId;
 
   // Remove the resource if it exists
   let hook = await this.Hook.load({
```

Comment:
No need to load it...
Just do:

`this.Hook.remove({hookGroupId, hookId}, true /* ignore if not exists*/ );`

If hook doesn't exist the operation is idempotent by saying you deleted it... Otherwise, retries wouldn't be sane.